doc/common-issues.rst

```diff
@@ -0,0 +1,98 @@
+Frequently Encountered Issues
+=============================
+Also see the `TLS Examples <https://pymongo.readthedocs.io/en/stable/examples/tls.html>`_.
+
+.. contents::
+
+Server reports wire version X, PyMongo requires Y
+-------------------------------------------------
+
+When one attempts to connect to a <=3.4 version server, PyMongo will throw the following error::
+
+	>>> client.admin.command('ping')
+	...
+	pymongo.errors.ConfigurationError: Server at localhost:27017 reports wire version 5, but this version of PyMongo requires at least 6 (MongoDB 3.6).
+
+This is caused by the driver being too new for the server it is being run against.
+To resolve this issue either upgrade your database to version >= 3.6 or downgrade to PyMongo 3.x which supports MongoDB >= 2.6.
+
+
+'Cursor' object has no attribute '_Cursor__killed'
+--------------------------------------------------
+
+On versions of PyMongo <3.9, when supplying invalid arguments the constructor of Cursor,
+there will be a TypeError raised, and an AttributeError printed to ``stderr``. The AttributeError is not relevant,
+instead look at the TypeError for debugging information::
+
+	>>> coll.find(wrong=1)
+	Exception ignored in: <function Cursor.__del__ at 0x1048129d8>
+	...
+	AttributeError: 'Cursor' object has no attribute '_Cursor__killed'
+	...
+	TypeError: __init__() got an unexpected keyword argument 'wrong'
+
+To fix this, make sure that you are supplying the correct keyword arguments.
+In addition, you can also upgrade to PyMongo >=3.9, which will remove the spurious error.
+
+
+MongoClient fails ConfigurationError
+------------------------------------
+
+MongoClient has historically accepted CodecOptions's members as keyword arguments,
+however this does not work for type_registry in PyMongo <3.8::
```

Comment:
This sentence should be updated now that were only talking about incorrect kwargs in general.